blog/2021/11/julia-1.7-highlights.md

```diff
@@ -0,0 +1,561 @@
++++
+mintoclevel = 2
+maxtoclevel = 2
+title = "Julia 1.7 Highlights"
+authors = "..."
```

Comment:
I presume this needs to be filled?
```suggestion
authors = "Jeff Bezanson, Jameson Nash, Ian Butterworth, Kristoffer Carlsson, Shuhei Kadowaki, Elliot Saba, Viral B Shah, Mosè Giordano, Simeon Schaub, Nicholas Bauer, Keno Fischer"
```
Or the `"..."` does something special?

Reply:
https://github.com/JuliaLang/www.julialang.org/blob/ca2f054294368ac0964b91cf353db61572eb51b5/previews/PR1419/blog/2021/11/julia-1.7-highlights/index.html#L1

it does expand automatically it seems

Reply:
Yea it was just a placeholder. I don't actually know who have authored the post.

Reply:
I went with the authors of the sections :stuck_out_tongue: